maxim/tests/test_together.py

```
@@ -0,0 +1,552 @@
+import os
+import time
```

Ruff (0.11.9)

`time` imported but unused

Remove unused import: `time`

(F401)